examples/msbr/msbr_endfb71_main.json

```diff
@@ -0,0 +1,27 @@
+{
+   "proc_input_file": "msbr_objects.json",
+   "dot_input_file": "msbr.dot",
+   "output_path": "./data",
+   "num_depsteps": 12,
+   "depcode": {
+       "codename": "serpent",
+       "exec_path": "sss2",
+       "template_input_file_path": "./msbr_endfb71.serpent",
+       "npop": 50,
+       "active_cycles": 20,
+       "inactive_cycles": 20,
```

Comment:
are these pop/active/inactive values just for testing?  They seem very low.

Reply:
Correct. They are so we make sure the model actually will run in saltproc (it doesn't right now as I detailed in issue #171)